packages/accounts-base/accounts_server.js

```diff
@@ -878,16 +878,17 @@ export class AccountsServer extends AccountsCommon {
     };
   };
 
-  // Using $addToSet avoids getting an index error if another client
-  // logging in simultaneously has already inserted the new hashed
-  // token.
   _insertHashedLoginToken(userId, hashedToken, query) {
     query = query ? { ...query } : {};
     query._id = userId;
     this.users.update(query, {
+      // Using $addToSet avoids getting an index error if another client
+      // logging in simultaneously has already inserted the new hashed
+      // token.
       $addToSet: {
         "services.resume.loginTokens": hashedToken
-      }
+      },
+      $set: { lastLoginAt: new Date() }
```

Comment:
I have decided to update the `lastLoginAt` here so that we don't have any new call to the DB.

Reply:
I like this feature! I'm also in favor of reducing the number of database operations. I'm just not sure whether this is the correct place to do it. For example, we do use `_insertHashedLoginToken` to generate a login link that we send _somewhere_. By doing that, we no longer know when someone actually logged in, but when did we generate the token.

Maybe there's a better way of doing that? If not, it'd be important to highlight it in the docs.

Reply:
That is why this is a draft PR, so that we can discuss. In here this is (according to my understanding) triggered when the user logs in. It is not perfect, but I think this is a good start. I'll try to do a bit more testing to see if this triggers when the user is re-logged/resumed automatically by Meteor instead of doing the action themselves. If people could test this with their own cases to see if it works for them or there is a better place to put it that would be great.

Reply:
Did some testing and this only triggers when user logs in via a login function, not when resuming. For the resumption I had to add a DB call above.